Ogra/Encodable.swift

```diff
@@ -135,6 +135,7 @@ extension JSON {
                 dict[key] = value.JSONObject()
 			}
 			return dict
+        default:                 return NSNull()
```

Comment:
We should list the type explicitly here. Argo recently added the `Bool` case, which is the one that we aren't handling. I've been meaning to add in `Bool` support to Ogra for a while now, but life has intervened. This may be the kick up the bum I need.